src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/openstack/OpenstackJCloudsProvider.java

```diff
@@ -59,7 +64,9 @@
 @Component
 public class OpenstackJCloudsProvider extends JCloudsProvider {
 
-    private final static String region = "RegionOne";
+    private static final Logger logger = Logger.getLogger(OpenstackJCloudsProvider.class);
+
+    private static final String REGION = "RegionOne";
 
     @Getter
     private final String type = "openstack-nova";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1170)